docs/coordinates/skycoord.rst

```diff
@@ -223,6 +222,27 @@ If the frame is not supplied then you will see a special ``ICRS``
 identifier.  This indicates that the frame is unspecified and operations
 that require comparing coordinates (even within that object) are not allowed.
 
+It is important to note that frame classes expect to receive Quantity-like objects with angular units. For example,
+
+  >>> import astropy
+  >>> from astropy import units as u
+  >>> from astropy.coordinates import SkyCoord, ICRS
+  >>> astropy.__version__
+  '3.1rc2'
+  >>> ra = 0
+  >>> deg = 0
+  >>> SkyCoord(ra, deg, unit=u.deg)
+  <SkyCoord (ICRS): (ra, dec) in deg
+      (0., 0.)>
+  >>> ICRS(ra, deg, unit=u.deg)
```

Comment:
Either mark this to be skipped by doctest or omit it from example.

Reply:
@pllim , I removed SkyCoord example and triple quoted ICRS example that wasn't working.